app.js

```diff
@@ -111,9 +123,11 @@ async function main() {
   /* Get Terms */
   log(await getTerms());
   /* Get Members */
-  log(await getMembers());
+  log(await getMembers('<SECTIONID>', '<TERMID>'));
```

Comment:
I noticed that gerMembers didn't define what it was looking for from @param

Reply:
Well spotted. Thanks